Gemfile

```diff
@@ -22,7 +22,7 @@ gem "default_value_for",              "~>3.0.2"
 gem "elif",                           "=0.1.0",    :require => false
 gem "fast_gettext",                   "~>1.1.0"
 gem "fog-google",                     "~>0.3.0",   :require => false
-gem "fog-vcloud-director",            "~>0.1.1",   :require => false
+gem "fog-vcloud-director",            "~>0.1.2",   :require => false
```

Comment:
I added the "suspended" state as it was not exposed previously: https://github.com/xlab-si/fog-vcloud-director/commit/083379c86bdeff55f30633385a8f8c25bbdc75ee